src/Robo/Plugin/Traits/NotifierTrait.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace Usher\Robo\Plugin\Traits;
+
+use GuzzleHttp\Client;
+use Robo\Result;
+
+/**
+ * Trait to provide notification functionality.
+ */
+trait NotifierTrait
+{
+    /**
+     * Notify Slack.
+     *
+     * @param string $username
+     *   The username to pass to Slack.
+     * @param string $text
+     *   The text string to pass to the Slack API.
+     *
+     * @see https://docs.tugboatqa.com/starter-configs/code-snippets/slack-integration/
+     */
+    public function notifySlack(string $username, string $text): void
+    {
+        // Verify that a Slack webhook URL was provided.
+        $slack_webhook_url = getenv('SLACK_WEBHOOK_URL');
+        if ($slack_webhook_url === false || $slack_webhook_url === '') {
+            $this->yell('Missing Slack Webhook URL from the "SLACK_WEBHOOK_URL" environment variable.');
```

Comment:
@adamzimmermann I created an "Usher" app in Slack and added a webhook that points to #sandbox. Hopefully, if you add it to a repo where you are testing this, it should work.

```
https://hooks.slack.com/services/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
```

If it works how we want, we would create a webhook for each repo and point it to the appropriate channel.

Reply:
We had a public method in the trait that Robo was trying to expose as a command. I simply had to make it protected. 🙈